components/org.wso2.carbon.identity.sts.passive.ui/src/main/java/org/wso2/carbon/identity/sts/passive/ui/PassiveRequestorConstants.java

```diff
@@ -65,6 +65,11 @@ public class PassiveRequestorConstants {
 
     public static final String RELYING_PARTY_REALMS = "realms";
 
+    public static final String ERROR_AUTHENTICATION = "Authentication Error!";
+
+    public static final String ERROR_MSG_LOGOUT_WREPLY_MISMATCH = "Error in validating the application's logout " +
+            "redirect URL against the configured redirect URL.";
```

Comment:
against the configured redirect URL->against the configured WReply Logout URL

Reply:
Updated with https://github.com/wso2-extensions/identity-inbound-auth-sts/pull/144/commits/e00442f85066f9b0db4d39b30440e1f1e383cb56